crates/ln-dlc-node/src/ln/config.rs

```diff
@@ -14,6 +14,8 @@ pub fn app_config() -> UserConfig {
             minimum_depth: 1,
             // There is no risk in the leaf channel to receive 100% of the channel capacity.
             max_inbound_htlc_value_in_flight_percent_of_channel: 100,
+            // the delay before the fund of a force closed channel can be claimed again
+            our_to_self_delay: 144,
```

Comment:
:wrench: This patch is not just "adding a test". Can you either rename the patch or split it in 2?